src/pixie_cli/install.sh

```diff
@@ -41,6 +34,17 @@ PIXIE_BANNER="
  |_|  |_|/_\_\|_|\___|
 "
 
+ARTIFACT_BASE_PATH="https://github.com/${GITHUB_REPO}/releases/download/release%2Fcli%2Fv"
+
+# Fetch latest version.
+LATEST_VERSION=""
+regex='\[CLI v(([0-9]+)\.([0-9]+)\.([0-9]+))\]'
+if [[ "$(curl -fsSL https://raw.githubusercontent.com/pixie-io/pixie/main/README.md)" =~ $regex ]]; then
```

Comment:
maybe grep for `<!--cli-latest-release-->` first, in case the pattern appears elsewhere in the readme. Also, could consider parsing the manifest instead of the readme, but that might be hard without `jq`